packages/compass-e2e-tests/tests/connection.test.ts

```diff
@@ -110,6 +115,127 @@ function generateIamSessionToken(): {
   };
 }
 
+async function assertCanReadData(
+  browser: CompassBrowser,
+  dbName: string,
+  collectionName: string
+): Promise<void> {
+  await browser.navigateToCollectionTab(dbName, collectionName, 'Documents');
+  await browser.waitUntil(async () => {
+    const text = await browser
+      .$(Selectors.DocumentListActionBarMessage)
+      .getText();
+    return !!/Displaying documents \d+ - \d+ of \d+/.exec(text);
```

Comment:
tiny nit, feel free to ignore, but since there is a regexp method that is specifically for returning booleans… :)

```suggestion
    return /Displaying documents \d+ - \d+ of \d+/.test(text);
```

Reply:
I can never remember regex method names and regex related methods on strings for some reason :)